source/brailleDisplayDrivers/handyTech.py

```diff
@@ -524,6 +525,20 @@ class Modular80(Modular):
 	numCells = 80
 
 
+class Activator(TimeSyncFirmnessMixin, AtcMixin, JoystickMixin, TripleActionKeysMixin, Model):
+	deviceId = MODEL_ACTIVATOR
+	numCells = 40
+	genericName = name = 'Activator'
+
+	def _get_keys(self):

```

Comment:
Could this please get a type hint

```suggestion
	def _get_keys(self) -> Dict[int, str]:
```